__tests__/app.test.js

```diff
@@ -172,14 +206,13 @@ describe("/api/articles", () => {
 
         })
     });
-    test("GET: 404 - responds with an error when passed an article_id that is valid but not present in our database", ()=>{
+    test("GET: 400 - responds with an error when passed an invalid order query", () => {
```

Comment:
I don't think I can see a test similar to this, but for an invalid sort_by? If you don't have one, you should add one please.